config/service_config.go

```diff
@@ -105,6 +106,20 @@ func NewServiceConfig(id string, context context.Context) *ServiceConfig {
 	}
 }
 
+// Get Random Ports with size
+func getRandomPorts(size int) []int {
```

Comment:
Do we need to check whether the port is used by other applications?

Reply:
maybe.........

Reply:
@flycash recheck pls